### PR TITLE
Fix phone display / usability

### PR DESCRIPTION
**Pull request by DanielHeath**
- Sort freecall/phone/mobile
- Don’t show TTY or fax lines
- Don’t hide the second contact option if there are only two.

### CODE REVIEW

File: src/components/ServicePane.js [hunk 1]
```diff
@@ -54,9 +54,11 @@ export default class ServicePane extends React.Component {
 
     render(): React.Element {
         var object = this.props.service;
-        var phones = _.filter(object.phones, (p) => p.kind != 'fax');
+        var filteredPhoneKinds = _(['fax', 'tty']);
```
Comment (danni) [inline src/components/ServicePane.js]: Use a `Set`.

Comment (DanielHeath) [inline src/components/ServicePane.js]: Done :)

**Comment by danni** [comment]
++

**Comment by danni** [comment]
++

**Pull request merged**